afs/settings.py

```diff
@@ -2,6 +2,7 @@
 Django settings for afs project.
 """
 
+from arches.arches.settings import BYPASS_CARDINALITY_TILE_VALIDATION
```

Comment:
@ryan86 Can you remove this import? It's actually already covered by the `import *` on line 10.